### DISCUSSION TITLE
Home Assistant Integration

### DISCUSSION THREAD

**Discussion by FinestDice17** [bemasher/rtlamr#219] [Ideas]
So I'm wondering if there is a way to integrate all this into Home Assistant OS running on a RPI4 with they're new energy platform which graphs everything very nicely. Thank you in advanced.

**Comment by Nils154** [comment]
Yes, there is an amridm2mqtt repository on GitHub. You will have to tweak it a bit to have the right units, the Home Assistant Energy platform currently only accepts kWh and a few others, but not gallons or CCFs. 

--
Nils Gokemeijer (Sent from iPhone)

**Reply by johntdyer** [reply]
https://github.com/mdegat01/addon-amr2mqtt
works pretty good for me
